content-repo/extra-docs/releases/20.9.0.md

```diff
@@ -0,0 +1,418 @@
+# Cortex XSOAR Content Release Notes for version 20.8.2 (98126)
+##### Published on 1 September 2020
+### New: EmailCommunication Pack v1.0.0
```

Comment:
```suggestion
### New: Email Communication Pack v1.0.0
```

Reply:
We should also add a pack description for this one as it is a "use case" pack. We can copy the description from the pack:

> Do you have to send multiple emails to end users? This content pack helps you streamline the process and automate updates, notifications and more.